src/platform/site-wide/user-nav/containers/Main.jsx

```diff
@@ -39,11 +39,109 @@ import {
 } from 'platform/site-wide/user-nav/actions';
 import { updateLoggedInStatus } from 'platform/user/authentication/actions';
 import { ACCOUNT_TRANSITION_DISMISSED } from 'platform/user/authentication/constants';
+import { VaModal } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import * as authUtilities from 'platform/user/authentication/utilities';
 import SearchHelpSignIn from '../components/SearchHelpSignIn';
 import AutoSSO from './AutoSSO';
 import { selectUserGreeting } from '../selectors';
+import recordEvent from '../../../monitoring/record-event';
+
+// remove this when organic experiment finishes
+const CreateLoginGovAccountModal = ({ visible = false, onClose }) => {
```

Comment:
Move this entire component to `src/platform/authentication/components`, it'll be easier to remove later on (ie we won't need additional approvers)